classes/form/edit_entry.php

```diff
@@ -27,6 +27,21 @@ public function definition() {
         $mform->addElement('text', 'name', get_string('entryname', 'mod_cpdlogbook'));
         $mform->setType('name', PARAM_TEXT);
 
+        $mform->addElement('float', 'hours', get_string('entryhours', 'mod_cpdlogbook'));
```

Comment:
There is a 'duration' type form element, which I think is a bit nicer - but to swap to this you need to change the schema as it will store the duration in seconds rather than hours so its needs to a larger store it. I think defaulting to one hour is probably a reasonable default.